docs/tutorials/light-up.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Use Object Detection to Turn Your Lights On"
+linkTitle: "Turn on Lights with Object Detection"
+weight: 49
+type: "docs"
+tags: ["camera", "vision", "detector", "python"]
+description: "How to turn a light on when your webcam sees a person."
+# Author: Hazal Mestci
+---
+
+This tutorial uses the Viam [Vision Service](/services/vision/) with your computer's built-in webcam to detect the presence of a person and turn a lamp on when you sit down at your desk.

```

Comment:
I think it still makes sense to add here that they are using a smart Kasa plug because it gets introduced later on abruptly. ("This project is a great place to start if you are new to building robots, and have a smart plug laying around.")

Reply:
Ah good point